main.js

```diff
@@ -1,42 +1,140 @@
 var twoName = document.querySelector('#two-name');
 var oneGuess = document.querySelector('#one-guess');
 var twoGuess = document.querySelector('#two-guess');
+var rangeErrorMessageOne = document.querySelector('#range-error-one');
+var rangeErrorMessageTwo = document.querySelector('#range-error-two');
 var submitButton = document.querySelector('#submit-button');
 var clearButton = document.querySelector('#clear-button');
 var resetButton = document.querySelector('#reset-button');
+// Latest Guess
 var oneChallenger = document.querySelector('#one-challenger');
 var twoChallenger = document.querySelector('#two-challenger');
 var oneLatestGuess = document.querySelector('#one-latest-guess');
 var twoLatestGuess = document.querySelector('#two-latest-guess');
 var oneFeedback =document.querySelector('#one-guess-feedback');
 var twoFeedback =document.querySelector('#two-guess-feedback');
-var minRange = document.querySelector('#min-range');
-var maxRange = document.querySelector('#max-range');
-var minSpan = document.querySelector('#min-span');
-var maxSpan = document.querySelector('#max-span');
-var updateButton = document.querySelector('#update-button');
-var errorIcon = document.querySelector('#range-error'); // rename ID from error to range-error in HTNL, CSS, JS
+// Winner Card
+var winnerBox = document.querySelector('#winner-card');
+var outcomeNameOne = document.querySelector('#outcome-name-one');
+var outcomeNameTwo = document.querySelector('#outcome-name-two');
 var winnerName = document.querySelector('#winner-name');
+var summaryGuesses = document.querySelector('#summary-guesses');
+var closeButton = document.querySelector('#close-winner-output');
+// var playerInputBox = document.querySelectorAll('.player-input-box');
+// var errorIcon = document.querySelector('#range-error');
 
-submitButton.disabled = true;
-clearButton.disabled = true;
-resetButton.disabled = true;
-updateButton.disabled = true;
-
+// Event Listeners
+// Set Range
+minRange.addEventListener('click', checkRange);
+minRange.addEventListener('keyup', checkRange);
+maxRange.addEventListener('click', checkRange);
+maxRange.addEventListener('keyup', checkRange);
+updateButton.addEventListener('click', updateRange);
+// Guess Form
+document.addEventListener('keyup', rangeErrorCheckOne);
+document.addEventListener('keyup', rangeErrorCheckTwo);
 document.addEventListener('keyup', enableSubmit);
 document.addEventListener('keyup', enableClear);
+document.addEventListener('keyup', enableReset);
 clearButton.addEventListener('click', clearInputs);
 submitButton.addEventListener('click', updateLatestGuess);
-updateButton.addEventListener('click', updateRange);
-document.addEventListener('keyup', checkRange);
+resetButton.addEventListener('click', resetGame);
+// Winner Card
+closeButton.addEventListener('click', closeWinnerOutput);
 
+// Statements
+submitButton.disabled = true;
+clearButton.disabled = true;
+resetButton.disabled = true;
+updateButton.disabled = true;
```

Comment:
Create a function that will disable all buttons.  There are one or two functions that disable all buttons.  This will reduce the overall Javascript size.

Reply:
Good call, let's not repeat ourselves!